spec/FrameSpec.js

```diff
@@ -0,0 +1,87 @@
+describe ('Frame', function () {
+
+  var frame;
+
+  beforeEach(function () {
+
+    frame = new Frame
+
+    });
+
+  describe ('Scoring', function () {
+    it ('returns 0 for two gutter balls', function() {
+      frame.enterPins(0);
+      frame.enterPins(0);
+      expect(frame.getScore()).toEqual(0);
+    });
+
+    it ('returns score for two scoring rolls', function() {
+      frame.enterPins(1);
+      frame.enterPins(2);
+      expect(frame.getScore()).toEqual(3);
+    });
+
+    it ('scores 0 if a strike is thrown', function() {
```

Comment:
I hadn't thought about using proper bowling notation for strikes and spares, what advantages/disadvantages does this offer?